app/api/csv/csvLoader.ts

```diff
@@ -118,4 +118,36 @@ export class CSVLoader extends EventEmitter {
     return saved;
   }
   /* eslint-enable class-methods-use-this */
+
+  async loadTranslations(csvPath: string, translationContext: string) {
+    const file = importFile(csvPath);
+    const availableLanguages = ensure<LanguageSchema[]>(
+      (await settings.get()).languages
+    ).map((l: LanguageSchema) => ({ label: l.label, language: l.key }));
+
+    const intermidiateTranslation: any = {};
+
+    await csv(await file.readStream(), this.stopOnError)
+      .onRow(
+        async (row: CSVRow, _index: number): Promise<void> => {
+          Object.keys(row).forEach(lang => {
+            intermidiateTranslation[lang] = intermidiateTranslation[lang] || {};
+            intermidiateTranslation[lang][row.Key] = row[lang];
```

Comment:
I believe this was talked about in the standup

Reply:
Right, it was discussed in the standup and the decision, if I'm not wrong, was to restrict the key to the existing ones in the database,  but I can't see any change like that. 
cc. @RafaPolit

Reply:
I have restricted to only existing keys, I also have removed the test that checks whether new keys are added correctly because that does not apply any more. Also the context is always `System` because the user can only import when on  `System` context